help/C/Help_ch_Tools_Assistants.xml

```diff
@@ -3165,4 +3165,531 @@
+        transactions to account for the difference between the costs of buying
+        a security and the value received by selling it.
+      </para>
+
+      <para>To open the <guilabel>Lots in Account</guilabel> window, open the
+        security account register, then select
+        <menuchoice>
+          <guimenu>Actions</guimenu>
+          <guimenuitem>View Lots</guimenuitem>
+        </menuchoice>.
+      </para>
+
+      <figure pgwide="1">
+        <title>Lots in Account window for Security Capital Gains
+        </title>
+        <screenshot id="tool-lotsInAcctStk">
+          <mediaobject>
+            <imageobject role="html">
+            <imagedata fileref="figures/Help_Tool_LotsStk.png"
+              format="PNG" srccredit="Chris Good" width="510px"></imagedata>
+            </imageobject>
+            <imageobject role="fo">
+              <imagedata fileref="figures/Help_Tool_LotsStk.png"
+                         format="PNG" srccredit="Chris Good" ></imagedata>
+            </imageobject>
+
+            <textobject>
+              <phrase>Example of Lots in Account window for a security account
+              </phrase>
+            </textobject>
+
+            <caption><para>An image of the <emphasis>Lots in Account
+              </emphasis> window for a security account.</para>
+            </caption>
+          </mediaobject>
+        </screenshot>
+      </figure>
+
+      <sect3 id="tool-lots-cap-scr">
+        <title>Screen Elements</title>
+
+        <para><emphasis>Adjusting the Size of Screen Elements</emphasis>
+        </para>
+
+        <para>As well as adjusting the size of the whole window, it is
+          possible to adjust the proportions of the window elements both
+          horizontally and vertically.
+        </para>
+
+        <para>There are 2 bars for adjusting the window horizontally. One
+          between the <guilabel>Title</guilabel> and the <guilabel>Lots in
+          This Account</guilabel> panel, and the second between the <guilabel>
+          Splits free</guilabel> panel and the <guilabel>&gt;&gt;</guilabel>
+          and <guilabel>&lt;&lt;</guilabel> buttons.
+        </para>
+
+        <para>The window proportions may be adjusted vertically by
+          dragging the vertical bar located to the right of <guilabel>Show
+          only open lots</guilabel>.
+        </para>
+
+        <para>These resizing bars can be hard to see, so move the mouse
+          pointer around until it changes to the resize pointer (<guilabel>
+          &lt;&mdash;&gt;</guilabel> in the case of the horizontal bar), then
+          click and drag the bar.
+        </para>
+
+        <para>It is not possible to individually adjust the size of panel
+          columns.
+        </para>
+
+        <para><emphasis>Fields</emphasis></para>
+        <itemizedlist>
+          <listitem>
+            <para><guilabel>Title</guilabel> &mdash; Shows the title for the
```

Comment:
I suppose to be pedantically correct it should be &lt;guimenuitem&gt; instead of &lt;guilabel&gt;, but I think both tags render the same so it doesn't really matter for our purposes.